server/paths/assess.ts

```diff
@@ -24,9 +25,8 @@ export default {
     show: courseCaseListPath.path(':referralStatusGroup'),
   },
   moveToBuildingChoices: {
-    reason: {
-      show: moveToBuildingChoicesReason,
-    },
+    check: moveToBuildingChoicesCheck,
```

Comment:
We should use either move or transfer and use them consistently.

Also, `/move-to-building-choices/check` could just be `/move-to-building-choices` and then you could have the reason at `/move-to-building-choices/reason`